app/templates/components/upload-button.hbs

```diff
@@ -1,8 +1,14 @@
 <div class="col-sm-12" style="padding-bottom:30px;">{{!COL}}
-    {{fa-icon icon="cloud-upload" size=5}}
-    <h1 class="f-w-xl">Contribute your research</h1>
-    <h3 class="f-w-md" style="margin-left:auto; margin-right:auto;">You can upload a preprint or choose a file from an existing OSF file to tag as a preprint if you are already have an OSF account.</h3>
 
+    <div class="headers-div"> <!--headers div-->
+                    <h2>CONTRIBUTE</h2>
+                <h4>your research</h4>
```

Comment:
Why does this go from h2 to h4? If you just to control text size, CSS may be the way to go instead of heading tags.

(the latter is used to create an outline of the document, eg by some screen readers)